examples/How_to_handle_rate_limits.ipynb

```diff
@@ -218,10 +159,10 @@
     "\n",
     "@retry(wait=wait_random_exponential(min=1, max=60), stop=stop_after_attempt(6))\n",
     "def completion_with_backoff(**kwargs):\n",
-    "    return openai.Completion.create(**kwargs)\n",
+    "    return openai.ChatCompletion.create(**kwargs)\n",
```

Comment:
Ah, actually, too many changes in flight since dev-day haha. 

Can we update this to use our latest [python SDK](https://github.com/openai/openai-python)? `client.chat.completions.create`

Reply:
It is updated to latest python sdk!